lib/Frontend/CompilerInvocation.cpp

```diff
@@ -1477,6 +1521,10 @@ bool CompilerInvocation::parseArgs(
   setDefaultPrebuiltCacheIfNecessary(FrontendOpts, SearchPathOpts,
                                      LangOpts.Target);
 
+  // Now that we've parsed everything, setup some inter-option-dependent state.
+  setIRGenOutputOptsFromFrontendOptions(IRGenOpts, FrontendOpts);
+  setBridgingHeaderFromFrontendOptions(ClangImporterOpts, FrontendOpts);
```

Comment:
`setBridgingHeaderFromFrontendOptions` likely has the same issue for me as `setIRGenOutputOpts...`